.github/workflows/airflow-plugin.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           directory: .
           fail_ci_if_error: false
```

Comment:
we also need to tweak the `if` clause in the above two steps

Reply:
do you mean upload test results artifact for all versions with flags matching the test case details?

Reply:
I think the upload test results artifact is fine to leave as is, but the codecov should upload `if: {{ always() }}`

Reply:
okay, so will upload test results only for latest version and codecove for all.